x/ccv/provider/keeper/hooks.go

```diff
@@ -167,3 +190,68 @@ func (h Hooks) AfterValidatorBeginUnbonding(_ sdk.Context, _ sdk.ConsAddress, _
 func (h Hooks) BeforeDelegationRemoved(_ sdk.Context, _ sdk.AccAddress, _ sdk.ValAddress) error {
 	return nil
 }
+
+//
+// gov hooks
+//
+
+// AfterProposalSubmission - call hook if registered
```

Comment:
Can you put this into a file called `gov_hooks.go`?

Reply:
> Nit: Can you put this into a file called gov_hooks.go?
It'd imply to also create a `staking_hooks.go` file to be consistent.
With hooks from only two modules, I think it's still manageable to keep them in a single file.